src/ds/d400/d400-device.cpp

```diff
@@ -1081,6 +1103,32 @@ namespace librealsense
         //TODO: Implement
     }
 
+    // Check if need change camera name due to number modifications on one device PID.
+    void update_device_name(std::string& device_name, const ds::d400_caps cap)
+    {
+        std::regex old_name_reg("D455");
+
+		switch (cap)
+		{
+		    case ds::d400_caps::CAP_IR_FILTER:
+                device_name += "F"; // Adding "F" to end of device name if it has IR filter.
+                break;
+
+		    case ds::d400_caps::CAP_IP65:
+                device_name = std::regex_replace(device_name, old_name_reg, "D456"); // Change device name from D455 to D456.
```

Comment:
Notice you have alignment issues on several lines here.
Please use 'clang-format', (if you don't know how ask..)

Reply:
May you send me a link to read about clang-format?

Reply:
Sent